src/test/java/org/sefglobal/scholarx/service/MentorServiceTest.java

```diff
@@ -87,7 +87,7 @@ void applyAsMentee_withValidData_thenReturnCreatedData()
                 .findByProfileIdAndProgramId(anyLong(), anyLong());
         doReturn(Optional.empty())
                 .when(menteeRepository)
-                .findByProfileIdAndAppliedMentorId(anyLong(), anyLong());
+                .findByProgramIdAndProfileId(anyLong(), anyLong())
```

Comment:
```suggestion
                .findByProgramIdAndProfileId(anyLong(), anyLong());
```
@cnonnet  I think you missed the semi colon.